### ISSUE TITLE
Fullfillments for 2022-07 version

### ISSUE THREAD

**Issue by damix1984** [phpclassic/php-shopify#268]
in version 2022-07 the orders/$id/fulfillments.json API is no more present.

We need to use the $client->Fulfillments making Fulfillment not a Child resource for orders but a Resource itself, to be able to call fulfillments.json endpoint.

This is related to https://github.com/phpclassic/php-shopify/issues/264

**Comment by fzoccara** [comment]
Hi there,
how come the fullfillment api are gone with out a backward compatibility? they are precious to me too..
Please put them back in version 2022-07 and in the following versions.

Best.
Francesco.

**Comment by nVuln** [comment]
@damix1984 yup, I mean they removed order->fulfillment api, but if you still want to use it, just re-defined api version to 2022-07, this is temporarily until you totally migrate your code to their new api version

**Comment by jeroendelau** [comment]
I've done some work towards this goal, although I'm not sure this covers all scenarios.

https://github.com/phpclassic/php-shopify/pull/278

**Comment by tonytoms** [comment]
@damix1984 : 
Strangely, i got the below response from Shopify

> Another solution that you could consider is to use the REST Admin API, which provides more granular control over the fulfillment process. You can use the POST /admin/api/2022-01/orders/#{id}/fulfillments.json endpoint to update the fulfillment

Does that mean admin/api/2021-01/orders/$id/fulfillments.json is not the one that is getting deprecated?

**Comment by damix1984** [comment]
> @damix1984 : Strangely, i got the below response from Shopify
> 
> > Another solution that you could consider is to use the REST Admin API, which provides more granular control over the fulfillment process. You can use the POST /admin/api/2022-01/orders/#{id}/fulfillments.json endpoint to update the fulfillment
> 
> Does that mean admin/api/2021-01/orders/$id/fulfillments.json is not the one that is getting deprecated?

It is deprecated and removed since 2022-07 version, so it still works in 2022-01, but no more since 2022-07
